### DISCUSSION TITLE
lotus daemon --import-snapshot ERROR

### DISCUSSION THREAD

**Discussion by Surasit20** [filecoin-project/lotus#9068] [General Q&A] [answered]
lotus version 1.13.2-rc4+mainnet+git.e05fdf892
This error:
ERROR: failed to start json-rpc endpoint: could not listen: listen tcp4 127.0.0.1: 1234: bind: address already in use

**Comment by LexLuthr** [comment, marked as answer]
Port 1234 is already in use. Please other process using that port. You can either terminate them or use a new port by specifying `lotus daemon --api <port>`
You seem to be using a really old version of lotus. Please note that it will not work after network upgrade. I would suggest using v1.16.1

**Reply by Surasit20** [reply]
Thank you for your answer.
